src/tests/test_pyodide.py

```diff
@@ -118,3 +120,147 @@ def eval_code(code, ns):
     )
     assert selenium.run("x = 99; 5") == [3, 5]
     assert selenium.run("7") == [99, 7]
+
+
+def test_mount_object(selenium):
+    result = selenium.run_js(
+        """
+        function x1(){
+            return "x1";
+        }
+        function x2(){
+            return "x2";
+        }
+        function y(){
+            return "y";
+        }
+        let a = { x : x1, y, s : 3, t : 7};
+        let b = { x : x2, y, u : 3, t : 7};
+        pyodide.mountPackage("a", a);
+        pyodide.mountPackage("b", b);
+        return pyodide.runPython(`
+            from a import x
+            from b import x as x2
+            result = [x(), x2()]
+            import a
+            import b
+            result += [a.s, dir(a), dir(b)]
+            # clean up to prevent this test
```

Comment:
Is this comment incomplete?

Reply:
I think this stuff is no longer necessary.